app/src/test/java/org/simple/clinic/drugs/EditMedicinesUiRendererTest.kt

```diff
@@ -97,4 +98,19 @@ class EditMedicinesUiRendererTest {
     verify(ui).showDoneButton()
     verifyNoMoreInteractions(ui)
   }
+
+  @Test
+  fun `when open intention is to refill medicine, then show refill medicine button `() {
+    // given
+    val defaultModel = EditMedicinesModel.create(patientUuid, RefillMedicine)
+    val prescribedDrugRecords = emptyList<PrescribedDrug>()
```

Comment:
Let's add couple of prescribed drugs here, please? Since the refill open intention is only possible when the prescribed drugs are present when opening. So we want to recreate the actual conditions as well.